### ISSUE TITLE
Replace “/var/run” with “/run” everywhere

### ISSUE THREAD

**Issue by DemiMarie** [QubesOS/qubes-issues#6315]
**The problem you're addressing (if any)**

Using `/var/run` in `tmpfiles.d` causes warnings.  On systemd-based Linux distributions, using it elsewhere is merely unnecessarily verbose.

**Describe the solution you'd like**
Replace `/var/run` with `/run` :smile:.

**Where is the value to a user, and who might that user be?**
Users won’t get a warning message from `systemd-tmpfilesd.service`

**Describe alternatives you've considered**
N/A

**Additional context**

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

**Comment by DemiMarie** [comment]
On further thought, this might not be a good idea outside of systemd-related code.  In particular, `/run` isn’t portable to other \*nix variants, such as OpenBSD, whereas `/var/run` seems to be.

**Comment by marmarek** [comment]
Yes, I think it's reasonable to limit the change to systemd-related code (unit files, tmpfiles.d/*, pidfiles etc). I don't think `/var/run` -> `/run` symlink is going anywhere in a foreseeable future.

**Comment by unman** [comment]
On Tue, Dec 29, 2020 at 04:31:11AM -0800, Marek Marczykowski-G??recki wrote:
> Yes, I think it's reasonable to limit the change to systemd-related code (unit files, tmpfiles.d/*, pidfiles etc). I don't think `/var/run` -> `/run` symlink is going anywhere in a foreseeable future.
> 

This triggers warning on deb package installs - good to see it go.
